### PR TITLE
text in wild ocr added

### PR DESCRIPTION
**Pull request by 17sushmita**
natural_photo_se_text.py added

### CODE REVIEW

File: freshlybuiltimagebol/natural_photo_se_text.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            startY = int(endY - h)
+
+            rects.append((startX, startY, endX, endY))
+            confidences.append(scoresData[x])
+
+    return (rects, confidences)
+
+
+
+
```
Comment (vishal2612200) [inline freshlybuiltimagebol/natural_photo_se_text.py:47]: remove extra space from the file

File: freshlybuiltimagebol/natural_photo_se_text.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
+from imutils.object_detection import non_max_suppression
+import numpy as np
+import pytesseract
+import argparse
+import cv2
```
Comment (vishal2612200) [inline freshlybuiltimagebol/natural_photo_se_text.py:5]: use from to import specific functions.

File: freshlybuiltimagebol/natural_photo_se_text.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        print("OCR TEXT : ",text)
+
+        text = "".join([c if ord(c) < 128 else "" for c in text]).strip()
+        #fbi.ShabdDhwani.shabd_se_dhwani(text,'en',"out.mp3")
+        #playsound('out.mp3')
+        cv2.rectangle(output, (startX, startY), (endX, endY),
+            (0, 0, 255), 1)
+        cv2.putText(output, text, (startX, startY - 20),
+            cv2.FONT_HERSHEY_SIMPLEX, 0.5, (255, 0, 0), 2)
+    cv2.imwrite('images/out.jpg',output)
```
Comment (vishal2612200) [inline freshlybuiltimagebol/natural_photo_se_text.py:112]: this name or location should be given by the developer.

File: freshlybuiltimagebol/natural_photo_se_text.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
+from imutils.object_detection import non_max_suppression
+import numpy as np
+import pytesseract
+import argparse
+import cv2
+from playsound import playsound
+import freshlybuiltimagebol as fbi
+
```
Comment (vishal2612200) [inline freshlybuiltimagebol/natural_photo_se_text.py:8]: make a single class for this py file. make these functions as class methods.

**Comment by 17sushmita** [comment]
natural_photo_se_text.py Updated

File: freshlybuiltimagebol/natural_photo_se_text.py [hunk 5]
```diff
@@ -1,120 +1,111 @@
...
 
+        (newW, newH) = (width, height)
+        rW = origW / float(newW)
+        rH = origH / float(newH)
 
-def text_pehchano(image,min_confidence=0.85,width=320,height=320,padding=0.00):
-    east='frozen_east_text_detection.pb'
-    orig = image.copy()
-    (origH, origW) = image.shape[:2]
+        image = cv2.resize(image, (newW, newH))
```
Comment (vishal2612200) [inline freshlybuiltimagebol/natural_photo_se_text.py:43]: use from statement ' from cv2 import resize' then use 'resize(image, (newW, newH))'

File: freshlybuiltimagebol/natural_photo_se_text.py [hunk 6]
```diff
@@ -1,120 +1,111 @@
 from imutils.object_detection import non_max_suppression
 import numpy as np
```
Comment (vishal2612200) [inline freshlybuiltimagebol/natural_photo_se_text.py:2]: use from statement in place of import

File: freshlybuiltimagebol/natural_photo_se_text.py [hunk 7]
```diff
@@ -1,120 +1,111 @@
...
+                startX = int(endX - w)
+                startY = int(endY - h)
+
+                rects.append((startX, startY, endX, endY))
+                confidences.append(scoresData[x])
+
+        return (rects, confidences)
+    
+    def text_pehchano(image,min_confidence=0.85,width=320,height=320,padding=0.00):
+        east='frozen_east_text_detection.pb'
```
Comment (vishal2612200) [inline freshlybuiltimagebol/natural_photo_se_text.py:37]: please upload the used model under the 'ml_model' folder name. then update the model location here.

File: freshlybuiltimagebol/natural_photo_se_text.py [hunk 8]
```diff
@@ -1,120 +1,111 @@
...
-            rects.append((startX, startY, endX, endY))
-            confidences.append(scoresData[x])
-
-    return (rects, confidences)
-
-
-
+from .text_bol_uthega import ShabdDhwani
+class NaturalPhotoShabd:
+    def result_decode_kro(scores, geometry,min_confidence):
```
Comment (vishal2612200) [inline freshlybuiltimagebol/natural_photo_se_text.py:8]: decode is english name,  replace it with Hindi keyword

Comment (vishal2612200) [reply freshlybuiltimagebol/natural_photo_se_text.py:8]: Also, add a description about the function functionality as a multiline comment.

**Comment by KapilBansal320** [comment]
error
Error while running text_pehchano functions. I use cv2 to read the image file

**Comment by KapilBansal320** [comment]
import NaturalPhotoShabd in init file. Also, change either the location or address of your model to 'freshlybuiltimagebol/models/frozen_east_text_detection.pb'

**Comment by 17sushmita** [comment]
It was a path error. I have updated the file.

**Comment by KapilBansal320** [comment]
Squash your commits in a single commit

**Comment by 17sushmita** [comment]
Updated README.md for natural_photo_se_text.py

**Pull request merged**